intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -295,7 +295,7 @@ \section{Monitoring and managing your job(s)}
 To get the status information on your job:
 
 \begin{prompt}
-%\shellcmd{qstat $<$jobid$>$}%
+%\shellcmd{qstat <{}jobid>{}}%
```

Comment:
Same here, use `qstat 12345`, mention to replace the `12345` with the actual job ID